src/components/Toggle/Toggle.module.scss

```diff
@@ -0,0 +1,77 @@
+@import "../../styles/states";
+@import "../../styles/global-css-settings";
+
+/* in the new toggle we drew. */
+
+/* When the hidden checkbox will be focused by keyboard navigation events, the toggle appearance will reflect it */
+
+.input:focus-visible ~ .toggle,
+.input.focusVisible ~ .toggle {
+  @include focus-style-css();
+}
+
+/* In order to allow accessibility, all operations will be performed on the hidden checkbox and be reflected */
```

Comment:
all comments become a mess, let's fix it before moving forward. The documentation here is importent because we did something a bit complex here.

Reply:
Fixed